concerned-coyotes/earlyinternet/dashboard/views.py

```diff
@@ -53,3 +56,20 @@ def index(request):
     return render(request=request,
                   template_name='dashboard.html',
                   context=context)
+
+
+def update_location(request: HttpRequest) -> HttpResponse:
+    """Updates the user models geolocation with location
+    from url parameters and redirects to dashboard page."""
+
+    longitude = float(request.GET.get("longitude"))
+    latitude = float(request.GET.get("latitude"))
+
+    # Update user model
+    if longitude and latitude:
```

Comment:
Not necessary to check with the code above. float() will either return a valid float or an exception